lib/test/fixtures/getDataContractFixture.js

```diff
@@ -0,0 +1,67 @@
+const DashPlatformProtocol = require('@dashevo/dpp');
```

Comment:
It this for a functional test? In unit tests, we can operate with any buffer.

Reply:
Yes. It's for functional tests basically. They are not ready yet.